garak/_config.py

```diff
@@ -253,3 +255,29 @@ def parse_plugin_spec(
             plugin_names.remove(plugin_to_skip)
 
     return plugin_names, unknown_plugins
+
+def load_translator(translation_service: str="", classname: str="") -> object:
```

Comment:
why is this in garak._config ?

Reply:
I follow the [advice](https://github.com/leondz/garak/pull/943#discussion_r1793559331). 
But If I misunderstand it, please let me know your recommend way to implementation.